src/tests/ftest/util/server_utils.py

```diff
@@ -247,6 +246,8 @@ def get_sub_command_class(self):
             #   scan     Scan SCM and NVMe storage attached to local server
             if self.sub_command.value == "prepare":
                 self.sub_command_class = self.PrepareSubCommand()
+            elif self.sub_command.value == "scan":
+                self.sub_command_class = self.ScanSubCommand()
```

Comment:
(pylint-redefined-variable-type) Redefinition of self.sub_command_class type from util.server_utils.DaosServerCommand.StorageSubCommand.PrepareSubCommand to util.server_utils.DaosServerCommand.StorageSubCommand.ScanSubCommand